spk/tt-rss/src/bin/php

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
The point of this script is to be reused by tt-rss when it tries to launch a PHP process.
Since the current Synology installation requires some extensions to be manually activated and since the same must be done when launching the update daemon might as well create two new files:

* `bin/php` which would redirect "automatically" to the proper installation
* `bin/php-settings.sh` which would be sourced to define what's need to launch a valid php command